setup.py

```diff
@@ -59,8 +50,10 @@
       download_url = 'https://github.com/RDFLib/sparqlwrapper/releases',
       platforms = ['any'],
       packages = ['SPARQLWrapper'],
-      requires = _requires,
```

Comment:
so, if I got it right, the overall patch would just to remove the `requires` entry, so we could keep the `install_required` generated from `requirements.txt`; is that true, @asergi?